manifests/server/database.pp

```diff
@@ -7,7 +7,7 @@
   Optional[String[1]]                           $backend         = undef,
   Optional[String[1]]                           $rootdn          = undef,
   Optional[String[1]]                           $rootpw          = undef,
-  Optional[String[1]]                           $initdb          = undef,
+  Optional[Boolean]                             $initdb          = undef,
```

Comment:
I find `Optional` strange for Booleans, because isn't `undef` equivalent to `false`? Seems like the default is `true` in the type. Maybe remove `Optional` and default it to `true`?

Reply:
Yes, this is indeed over-complicated and I was trying to decipher this.
Defaulting to true should be the right thing to do.

Reply:
I had to reverse this because the default value actually depend on the backend.